frontend/components/impact-chart/types.ts

```diff
@@ -0,0 +1,5 @@
+export type ImpactChartProps = {
+  impact: number[];
+  labels: JSX.Element[] | string[];
```

Comment:
Instead of receiving the labels, the component itself could fetch them using the enums endpoint. I think that would make the component more self contained, especially because we'll reuse it in several places.